app-modules/knowledge-base/src/Filament/Resources/KnowledgeBaseItemResource/Pages/ListKnowledgeBaseItems.php

```diff
@@ -79,25 +81,40 @@ public function table(Table $table): Table
                     ->translateLabel()
                     ->searchable()
                     ->sortable(),
+                TextColumn::make('category.name')
+                    ->label('Category')
+                    ->translateLabel()
```

Comment:
Let's remove the `translateLabel` here and everywhere else in this page. We are getting rid of that as it is not needed.